tests/lsdb/catalog/test_catalog.py

```diff
@@ -0,0 +1,33 @@
+import dask.dataframe as dd
+import pandas as pd
+from hipscat.pixel_math import HealpixPixel
+
+
+def test_catalog_repr_equals_ddf_repr(small_sky_order1_catalog):
+    assert (
+        repr(small_sky_order1_catalog) == repr(small_sky_order1_catalog._ddf)
+    )
+
+
+def test_catalog_html_repr_equals_ddf_html_repr(small_sky_order1_catalog):
+    assert (
+        small_sky_order1_catalog._repr_html_()
+        == small_sky_order1_catalog._ddf._repr_html_()
+    )
+
+
+def test_catalog_compute_equals_ddf_compute(small_sky_order1_catalog):
+    pd.testing.assert_frame_equal(
+        small_sky_order1_catalog.compute(), small_sky_order1_catalog._ddf.compute()
+    )
+
+
+def test_get_catalog_partition_gets_correct_partition(small_sky_order1_catalog):
+    for _, row in small_sky_order1_catalog.hc_structure.get_pixels().iterrows():
```

Comment:
I see this too much and it grosses me out. Maybe the `hc.Catalog.get_pixels()` method should not be here, or should be replaced by `get_healpix_pixels` that returns a list of `HealpixPixel`. Yet again, something that can wait.

Though I think that the lsdb catalog should also offer a `get_healpix_pixels` convenience method (even if it's just wrapping the one on `hc.Catalog`).

Reply:
Made an issue for this #21